apps/graph-api/src/api.service.ts

```diff
@@ -79,7 +79,7 @@ export class ApiService implements BeforeApplicationShutdown {
     const job = await this.graphChangeRequestQueue.add(`Request Job - ${data.referenceId}`, data, {
       jobId: data.referenceId,
     });
-    this.logger.debug(job);
+    this.logger.debug(JSON.stringify(job));
```

Comment:
Sometimes `this.logger.debug('Some message', obj)` works without having to invoke JSON.stringify. Worth a test.

Reply:
I think this was working before we added `transform: true,` this changes the raw javascript object into the object and somehow it prints object when logged directly